src/styles/_catalog.scss

```
@@ -1,19 +1,35 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
 
-  max-width: 944px;
+$card-width: 200px;
+
+.catalog {
+  gap: 48px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-width);
+  justify-content: center;
+  // max-width: 944px;
   margin: 0 auto;
 
+  @media(max-width: 1023.98px) {
```

no necessary magic decimal numbers
```suggestion
  @media(max-width: 1023px) {
```